src/afterSuccess.sh

```diff
@@ -0,0 +1 @@
+bash <(curl -s https://copilot.blackducksoftware.com/ci/travis/scripts/upload)
```

Comment:
I think this URL will not be the same, read the documentation at https://copilot.blackducksoftware.com/

Reply:
```
Setting up CoPilot requires modifying your workflow file (a file in .github/workflows/). Once you commit the change and your next build succeeds, you will find your results on this page! Please note that your build must be passing in order for CoPilot to generate results.

First, if you are not running in a Java environment, you'll have to setup Java:

- name: Set up Java (CoPilot)
      uses: actions/setup-java@v1
      with:
        java-version: 1.8
The following can then be added after your Java setup block to enable results:

- name: Upload to CoPilot
      if: github.event_name == 'push' || github.event_name == 'pull_request'
      run: bash <(curl -s https://copilot.blackducksoftware.com/ci/githubactions/scripts/upload)
One last thing – in order to to see which components were added or removed in pull requests, your CI workflow needs to be running builds including this step for both push and pull_request events.
```

Reply:
![image](https://user-images.githubusercontent.com/21307950/101460536-102cd180-397d-11eb-91c5-8b3eba426c0c.png)

Reply:
https://github.com/BlackDuckCoPilot?utf8=%E2%9C%93&tab=repositories&q=githubactions